tony-history-server/build.gradle

```diff
@@ -0,0 +1,41 @@
+apply plugin: "play"
+apply plugin: "idea"
+
+model {
+    components {
+        play {
```

Comment:
Please use two-space indentation in this build.gradle to be consistent with our other Gradle files.